test/unit/spec/common/jobs-Spec.js

```diff
@@ -419,7 +419,7 @@ define([
                 };
             });
         });
-        return { byId: outputJobs };
+        return outputJobs;
```

Comment:
alter the input for the functions that were changed from using `exec.jobs` to `exec.jobs.byId`